airbyte-integrations/connectors/source-stripe/source_stripe/schemas/persons.json

```diff
@@ -0,0 +1,362 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Persons Schema",
+  "additionalProperties": true,
+  "type": ["null", "object"],
+  "properties": {
```

Comment:
It looks like we are missing some properties. If we can add the following just to ensure full compatibility: `phone, email, address_kana, address_kanji, first_name_kana, first_name_kanji, full_name_aliases, gender, id_number_secondary_provided, maiden_name, nationality, political_exposure, registered_address`